enterprise/backend/src/metabase_enterprise/serialization/v2/seed_entity_ids.clj

```diff
@@ -34,13 +33,14 @@
 (defn- make-table-name->model
   "Create a map of (lower-cased) application DB table name -> corresponding Toucan model."
   []
-  (into {} (for [model (descendants :toucan1/model)
-                 :when (models/model? model)
-                 :let  [table-name (some-> model t2/table-name name)]
-                 :when table-name
-                 ;; ignore any models defined in test namespaces.
-                 :when (not (str/includes? (namespace model) "test"))]
-             [table-name model])))
+  (into {}
+        (for [model (concat (descendants :toucan1/model) (descendants :metabase/model))
+              :when (mdb.u/toucan-model? model)
+              :let  [table-name (some-> model t2/table-name name)]
+              :when table-name
+             ;; ignore any models defined in test namespaces.
+              :when (not (str/includes? (namespace model) "test"))]
```

Comment:
I'm wondering whether we need to care about ignoring models defined in test namespaces with toucan2 models. Could we remove this `:when` expression altogether?

Reply:
I don't find it necessary, I could see we make a wrapper for `t2/table-model` that we automatically derive `:metabase/model` or it's automatically done in toucan2 in the future.